js/components/form/daterange-picker.vue

```diff
@@ -15,25 +15,25 @@
         v-el:start-input :placeholder="_('Start')"
         @focus="onFocus"
         :required="required"
-        :value="start_value"
+        :value="start_value|dt date_format ''"
         :readonly="readonly">
     <span class="input-group-addon">{{ _('to') }}</span>
     <input type="text" class="input-sm form-control"
         v-el:end-input :placeholder="_('End')"
         @focus="onFocus"
         :required="required"
-        :value="end_value"
+        :value="end_value|dt date_format ''"
         :readonly="readonly">
     <div class="dropdown-menu dropdown-menu-right">
         <calendar :selected="current_value"></calendar>
     </div>
     <input type="hidden" v-el:start-hidden
-        :id="field.id + '-start'"
-        :name="field.id + '-start'"
+        :id="field.id + '.start'"
```

Comment:
Because the dot is made for nesting. All the fields with dot in names are nested into the json (as HTML forms are flat).
See in [the dataset form definition](https://github.com/opendatateam/udata/blob/dev/js/components/dataset/form.vue), spatial fields are nested so there are already dotted.
By the way I kept the IDs, but I think they are  useless with vue.js as there is the `v-el` directive